eng/helix.proj

```diff
@@ -149,16 +161,21 @@
     <Delete Condition="$(HelixTargetQueues.ToLowerInvariant().Contains('ubuntu'))"
       Files="@(WindowsFiles);@(OsxFiles)" />
 
+    <GenerateFileFromTemplate TemplateFile="$(MSBuildThisFileDirectory)testing\$(RunTestScript)"
+        OutputPath="$(TargetDir)$(RunTestScript)"
```

Comment:
This TargetDir is not correct.  In my PR this worked because I would generate in the context of each test project, so TargetDir would point to the output of the test.  Here you are running in the context of `helix.proj` so targetdir is the output directory of that one project.  I think you need to instead generate into the test's directory, or somehow copy there.